gns3fy/gns3fy.py

```diff
@@ -1897,3 +1898,25 @@ def restore_snapshot(self, name=None, snapshot_id=None):
 
         # Update the whole project
         self.get()
+
+    def arrange_nodes(self):
```

Comment:
The name should reference the shape of the arrangement, I put it in another comment but something like `arrange_nodes_geometrically`

Reply:
I think the method name is too large, maybe
arrange_nodes_circular(self, radius=120)